charts/aws-ebs-csi-driver/templates/_helpers.tpl

```diff
@@ -59,11 +59,24 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 Convert the `--extra-volume-tags` command line arg from a map.
 */}}
 {{- define "aws-ebs-csi-driver.extra-volume-tags" -}}
+{{- $evt := default .Values.extraVolumeTags .Values.controller.extraVolumeTags }}
 {{- $result := dict "pairs" (list) -}}
-{{- range $key, $value := .Values.extraVolumeTags -}}
+{{- range $key, $value := $evt -}}
 {{- $noop := printf "%s=%s" $key $value | append $result.pairs | set $result "pairs" -}}
 {{- end -}}
 {{- if gt (len $result.pairs) 0 -}}
 {{- printf "%s=%s" "- --extra-volume-tags" (join "," $result.pairs) -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Handle http proxy env vars
+*/}}
+{{- define "aws-ebs-csi-driver.http-proxy" -}}
+- name: HTTP_PROXY
+  value: {{ .Values.proxy.http_proxy | quote }}
+- name: HTTPS_PROXY
+  value: {{ .Values.proxy.http_proxy | quote }}
+- name: NO_PROXY
+  value: {{ .Values.proxy.no_proxy | quote }}
+{{- end -}}
```

Comment:
Don't know why verification pass, but we should have an empty line here.

Reply:
verify don't cares about yaml/tpl file, only gofmt :/

Reply:
I will add blank lines when I get a chance.

Reply:
@AndyXiangLi Blank lines added